src/map/script.cpp

```diff
@@ -23696,6 +23696,27 @@ BUILDIN_FUNC(round) {
 	return SCRIPT_CMD_SUCCESS;
 }
 
+BUILDIN_FUNC(getequiptradability) {
+	int i = -1, num;
+	TBL_PC *sd;
+
+	num = script_getnum(st, 2);
+
+	if (!script_charid2sd(3, sd)) {
+		script_pushint(st, 0);
+		return SCRIPT_CMD_FAILURE;
+	}
+
+	if (equip_index_check(num))
+		i = pc_checkequip(sd, equip_bitmask[num]);
+	if (i >= 0)
+		script_pushint(st, !(sd->inventory.u.items_inventory[i].expire_time || (sd->inventory.u.items_inventory[i].bound && !pc_can_give_bounded_items(sd))));
+	else
+		script_pushint(st, 1);
+
+	return SCRIPT_CMD_SUCCESS;
```

Comment:
before the line better to add some var if condition expression will have unexcepted behavior which will return to script value. like:
`script_pushint(st, status)`
Where status defined as 0 at the beginning and changed in if condition block to a proper value like 1 if everything success than at the end we will have only two values 0 and 1, 0 if everything is bad, 1 if is good.